src/pages/messaging/index.mdx

```diff
@@ -136,43 +144,28 @@ The LiveChat Messaging API adopts the following versioning convention:
 | **dev preview** | 3.4            | 3.4               |
 | **legacy**      | 3.1, 3.2       | 3.1, 3.2          |
 
-# Agent Chat API
-
-## Use cases
-
-The **Agent Chat API** allows for:
-
-- interactions with the chat as Agent (joining a chat, posting messages)
-- interact with the chat by Bot Agents (acting as Agents)
-- building a custom Agent App
-- browsing chat archives
-- banning Customers
-- and much more
-
-If you plan on using the **Agent Chat API** as **RTM API**, refer to [Agent Chat RTM API](./agent-chat-api/rtm-reference/). For the **Web API** usage, read the [Agent Chat Web API](./agent-chat-api/) document.
-
-## Chatting as Agent
-
-The number of chats an Agent can participate in is not limited. It can be configured in the Agent App by the license Owner or Admin.
-
-# Customer Chat API
-
-## Use cases
+# Agent side
 
-The **Customer Chat API** allows for:
+Agent-side communication allows you to chat not only chat as an agent **but also as a bot** (also known as bot agent). You can perform **simple actions** such as **browsing archives** or **banning customers** or **build your custom agent application** (a [sample one](/messaging/chat-sdk/#simple-agent) was built using Chat SDK).
```

Comment:
"allows you to chat not only chat as an agent" <--- double "chat" in the sentence